appeals/web/environment/config.js

```diff
@@ -51,6 +51,7 @@ export function loadConfig() {
 		AZURE_BLOB_EMULATOR_SAS_HOST,
 		AZURE_BLOB_USE_EMULATOR,
 		FEATURE_FLAG_BOAS_1_TEST_FEATURE,
+		FEATURE_FLAG_S78_FEATURES,
```

Comment:
Wouldn't `FEATURE_FLAG_S78` be enough? Seems overly redundant.

Reply:
we should also remove the old BOAS flag

Reply:
the old BOAS flag is used in the tests

Reply:
ok, we can remove once we get to fixing the e2e...